src/main/java/CarRacing/CarRacing.java

```diff
@@ -0,0 +1,55 @@
+package CarRacing;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+public class CarRacing {
+
+    public static final int BOUND = 10;
+    public static final int MIN_MOVE_VALUE = 4;
+
+    private static final Random random = new Random();
```

Comment:
`Random` 인스턴스도 `static final`를 활용하여 상수로 선언하셨으니, 이름을 대문자로 작성해보면 어떨까요?
```suggestion
    private static final Random RANDOM = new Random();
```

Reply:
수정해서 커밋 했습니다. 유의하도록 하겠습니다.